e2e/testsuite/grpc_query.go

```diff
@@ -0,0 +1,23 @@
+package testsuite
```

Comment:
thought it might be nice to have a separate file for core queries we support

Reply:
makes sense 👍